src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf

```diff
@@ -36,7 +36,7 @@
             </trans-unit>
             <trans-unit id="10">
                 <source>No session available, it either timed out or cookies are not enabled.</source>
-                <target>Ingen sesjon tilgjengeleg. Sesjonen er anten ikkje lenger gyldig, eller informasjonskapslar er ikke skrudd på i nettlesaren.</target>
+                <target>Ingen sesjon tilgjengeleg. Sesjonen er anten ikkje lenger gyldig, eller informasjonskapslar er ikkje skrudd på i nettlesaren.</target>
```

Comment:
"anten" or "enten"?

Reply:
Cool. Thank you.

Reply:
Official dictionary: https://ordbok.uib.no/perl/ordbok.cgi?OPP=anten&ant_bokmaal=5&ant_nynorsk=5&nynorsk=+&ordbok=nynorsk